src/aerie_cli/utils/configurations.py

```diff
@@ -0,0 +1,29 @@
+from aerie_cli.aerie_host import AerieHostConfiguration
+from aerie_cli.persistent import PersistentConfigurationManager
+
+import json
+
+def find_configuration(configuration: str) -> AerieHostConfiguration:
+    """Find a configuration by name or path.
+    
+    configuration (str): The name or path of a configuration. Paths should be to a configuration json file.
+
+    Returns:
+        AerieHostConfiguration
+    """
+    # search for configuration by name
+    for persistent_configuration in PersistentConfigurationManager.get_configurations():
+        if persistent_configuration.name != configuration:
+            continue
+        return persistent_configuration
```

Comment:
Could use `PersistentConfigurationManager.get_configuration_by_name`:

```suggestion
    try:
        return PersistentConfigurationManager.get_configuration_by_name(name)
    except ValueError:
        pass
```

Reply:
OK I *guess* best practices are to not use exceptions for control flow, so maybe my suggestion isn't very Pythonic.

Reply:
I think we should leave it as-is then

Reply:
Good call. I rescind my comment!